mappings/net/minecraft/world/entity/BasicEntityLookup.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_5578 net/minecraft/world/entity/BasicEntityLookup
```

Comment:
```suggestion
CLASS net/minecraft/class_5578 net/minecraft/world/entity/SectionedEntityLookup
```

Reply:
Not sectioned. @Gegy This look up composes of two independent objects: an id index and a section cache. Both are independently maintained by the caller of the constructor than by this object. Sectioned is only a minor feature of this.